src/config/dbconfig.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    database:'dd7d00vtg0nupq',
+    userName:'xpwcjxjjdcgvvy',
+    password:'645a48c0aee7c4843d1f47cfff1bf362f7c84d79ab5470012f44b8ffe3259004',
+    host:'ec2-174-129-33-107.compute-1.amazonaws.com',
+    port:'5432',
+    dialect:'postgres',
```

Comment:
Optional note:
If you're already hosting your app in AWS EC2, you might as well use a free tier RDS with postgres :)